docs/user_guide/rcm.md

```diff
@@ -4,7 +4,15 @@ RCM (short for ReCovery Mode) is a mode for the switch that allows nintendo to s
 
 -----
 
-### Emunand cfw (Recommended)
+!!!tip "Emunand vs Sysnand"
+	- Emunand stands for Emulated nand, this means that your entire nand 	   (system memory) will run off your sd card with custom firmware (CFW)     this nand is *completely seprate* from your sysnand.
+
```

Comment:
No blank lines between lists again please.

Reply:
When built looks fine, slight change to formatting, again, discussed in exelix's server.